tests/DFE.FindInformationAcademiesTrusts.CypressTests/cypress/e2e/regression/cookies.cy.ts

```diff
@@ -1,14 +1,90 @@
+import commonPage from "../../pages/commonPage";
 import cookiesPage from "../../pages/cookiesPage";
-describe("Testing the Cookie page and its options", () => {
+import navigation from "../../pages/navigation";
+
+describe('Cookie page and consent tests', () => {
 
     beforeEach(() => {
-        cy.login();
+        cy.login()
+        cy.visit('/cookies')
+    });
+
+    it('should check that both mandatory and optional cookies both exist after accepting optional cookies', () => {
+        cookiesPage
+            .acceptCookies()
+            .clickSaveChangesButton()
```

Comment:
Not sure it needs the save changes call since its already in the accept cookies method

Reply:
I've now split this out of the accept cookies method as I had intended this but left it in accidentally